domain-ee/ee-ep-merge-app/integration/mq_endpoint.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, name, exchange, req_queue, response_queue):
         self.index = 0
         self.auto_response_files = []
 
-        exchange_props = ExchangeProperties(name=exchange, auto_delete=True, passive_declare=False)
+        exchange_props = ExchangeProperties(name=exchange, auto_delete=False, passive_declare=False)
```

Comment:
For integration tests, the exchange must be explicitly declared because the exchange would not exist otherwise without `svc-bip-api` or `svc-bgs-api` having already declared it.